### PR TITLE
Halloween!!!

### PR DESCRIPTION
**Pull request by FreakinsPL**
This Halloween map is created by Freakins, this map adds some spooky decorations near the ship in Las Ventures.

Pictures:
sa-mp-460
sa-mp-461
sa-mp-463

### CODE REVIEW

**Comment by OttoRocket-LVP** [comment]
Looks really cool for the halloween season! Sorry I haven't seen it earlier.

File: data/decorations/halloween_decorations.json [hunk 1]
```diff
@@ -0,0 +1,603 @@
+{
+    "objects": [
```
Comment (OttoRocket-LVP) [inline data/decorations/halloween_decorations.json:2]: Nitty detail: Can you please resolve the tabbing here, much like the other json files.
Your code editor might support an autoformat option :)

File: javascript/features/decorations/decorations.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const kDecorationDirectory = 'data/decorations/';
 // The list of decorations available on Las Venturas Playground. Don't forget to add the setting.
 export const kDecorations = new Set([
     { filename: 'christmas_decorations.json', setting: 'decorations/christmas_decorations' },
+   { filename: 'halloween_decorations.json', setting: 'decorations/halloween_decorations' },
```
Comment (OttoRocket-LVP) [inline javascript/features/decorations/decorations.js]: nit: Your tabbing seems a bit off

File: javascript/features/settings/setting_list.js [hunk 3]
```diff
@@ -44,6 +44,7 @@ export const kSettingList = [
     /** Decoration related settings */
     new Setting('decorations', 'christmas_decorations', Setting.TYPE_BOOLEAN, false, 'Should The Strip be in a Christmas-y mood?'),
     new Setting('decorations', 'fantasy_island', Setting.TYPE_BOOLEAN, false, 'Is this the time for some fantasy?'),
+   new Setting('decorations', 'halloween_decorations', Setting.TYPE_BOOLEAN, false, 'Be afraid ... Be very afraid.'),
```
Comment (OttoRocket-LVP) [inline javascript/features/settings/setting_list.js]: Your tabbing seems a bit off here aswell.

Might be nice to note that the strip will be scary in the comment

**Review by OttoRocket-LVP — APPROVED** [review]
Once again, looks really nice 👍

File: javascript/features/decorations/decorations.js (hunk 2 above)
Comment (FreakinsPL) [reply javascript/features/decorations/decorations.js]: fixed

File: javascript/features/settings/setting_list.js (hunk 3 above)
Comment (FreakinsPL) [reply javascript/features/settings/setting_list.js]: fixed

File: data/decorations/halloween_decorations.json (hunk 1 above)
Comment (FreakinsPL) [reply data/decorations/halloween_decorations.json:2]: Fixed :)

Comment (OttoRocket-LVP) [reply data/decorations/halloween_decorations.json:2]: I think you forgot to push :(

Comment (OttoRocket-LVP) [reply data/decorations/halloween_decorations.json:2]: I mean, I think I looked with my ass

**Pull request merged**